app/app/users/[user_id]/page.tsx

```diff
@@ -0,0 +1,87 @@
+"use client";
+import Card from "react-bootstrap/Card";
+import Col from "react-bootstrap/Col";
+import Row from "react-bootstrap/Row";
+import Spinner from "react-bootstrap/Spinner";
+import Table from "react-bootstrap/Table";
+import AppBreadCrumb from "@/components/AppBreadCrumb";
+import { useToken } from "@/utils/auth";
+import { useUser } from "@/utils/users";
+import { User } from "@/types/users";
+import { formatDateTime } from "@/utils/formatters";
+
+type UserColumn = {
+  name: keyof User;
+  label: string;
+  renderer?: (user: User) => string;
+};
+
+// todo: this can be extended and used for the user list table column sorting, etc
+const COLUMNS: UserColumn[] = [
```

Comment:
UPPER_SNAKE_CASE for constants -- this is great. Thank you for helping me discover these conventions and for adopting them out of the gate.